packages/p2p/src/components/my-profile/payment-methods/payment-methods-list/payment-methods-list.jsx

```diff
@@ -43,37 +51,34 @@ const PaymentMethodsList = () => {
                         <Localize i18n_default_text='Add new' />
                     </Button>
                     <div className='payment-methods-list__list-container'>
-                        {sortPaymentMethodsListMethods([...my_profile_store.payment_methods_list_methods]).map(
-                            (payment_methods_list_method, key) => {
-                                const payment_methods_list = my_profile_store.advertiser_payment_methods_list.filter(
```

Comment:
please remove `advertiser_payment_methods_list` in `my_profile_store` too

Reply:
it still being used somewhere else. But, i will try to refactor it there

Reply:
we can leave it if still being used somewhere. let's not refactor it here since it's not within the scope of the card.